README.md

```diff
@@ -256,6 +256,38 @@ $writeApi->write($dataArray);
 $writeApi->write('h2o,location=west value=33i 15');
 ```
 
+#### Default Tags
+
+Sometimes is useful to store same information in every measurement e.g. `hostname`, `location`, `customer`. 
+The client is able to use static value, app settings or env variable as a tag value.
```

Comment:
Missing example how to use with app settings.

Reply:
Example is in subsection 'Via API'.